zerver/views/message_edit.py

```diff
@@ -172,12 +172,33 @@ def json_fetch_raw_message(
     request: HttpRequest,
     maybe_user_profile: Union[UserProfile, AnonymousUser],
     message_id: int = REQ(converter=to_non_negative_int, path_only=True),
+    include_full_message: bool = REQ(default=False, json_validator=check_bool),
```

Comment:
I went with the approach to have a boolean toggle including the whole message to avoid this endpoint failing for other clients. Also, it doesn't make sense to send full message object for message edit history endpoints.